src/main/scala/edu/ie3/mobsim/MobilitySimulator.scala

```diff
@@ -154,7 +154,7 @@ final class MobilitySimulator(
       availableChargingPoints: Map[UUID, Int],
```

Comment:
the `sendEvMovementsToSImona` has a pretty misleading name as we also get departures back. 
How about `exchangeEvMovementsWithSimona` ?

Reply:
Good call